### PR TITLE
ansible: remove debian7

### PR DESCRIPTION
**Pull request by danbev**
This is no longer in the CI so it should be alright to remove it.

Refs: https://github.com/nodejs/build/pull/2607

### CODE REVIEW

**Review by AshCripps — APPROVED** [review]

File: ansible/roles/baselayout/vars/main.yml [hunk 1]
```diff
@@ -4,7 +4,6 @@
 # variables shared in baselayout
 #
 
-git_no_binpkg: ['debian7']
```
Comment (richardlau) [inline ansible/roles/baselayout/vars/main.yml:7]: We'll need to check that removing `git_no_binpkg` doesn't cause an issue for https://github.com/nodejs/build/blob/f65a7e67e5b46bc735acbc1f4aff617715b2aa16/ansible/roles/baselayout/tasks/main.yml#L60

**Review by richardlau — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

File: ansible/roles/baselayout/vars/main.yml (hunk 1 above)
Comment (richardlau) [reply ansible/roles/baselayout/vars/main.yml:7]: Unfortunately removing it completely fails:
```
TASK [baselayout : check for a recent enough git] **********************************************************************************************************
fatal: [test-rackspace-debian8-x64-1]: FAILED! => {"msg": "The conditional check 'os in git_no_binpkg' failed. The error was: error while evaluating conditional (os in git_no_binpkg): 'git_no_binpkg' is undefined\n\nThe error appears to be in '/home/rlau/sandbox/github/build/ansible/roles/baselayout/tasks/main.yml': line 59, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: check for a recent enough git\n  ^ here\n"}
```

File: ansible/roles/baselayout/vars/main.yml [hunk 2]
```diff
@@ -4,7 +4,6 @@
 # variables shared in baselayout
 #
```
Comment (richardlau) [inline ansible/roles/baselayout/vars/main.yml:6]:
```suggestion

git_no_binpkg: [ ]
```

**Pull request merged**